citystreets/settings.py

```diff
@@ -21,7 +21,7 @@
 SECRET_KEY = "&fnx8u(w_wkn$by1)@z233)w)vg5u(@bmt=aaom^i6(dxvt_!4"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Is it a good idea to set this to false across the board? This might make debugging stuff locally difficult.

Reply:
Looks like debug is handle by django with static file.

https://stackoverflow.com/questions/5836674/why-does-debug-false-setting-make-my-django-static-files-access-fail

And we might need `WhiteNoise` to help us handle the static file while we turn off the debug.
Plus we might want to use another env var to set the debug to false during our production.
Sound like a huge work, so I think we should keep it in our product backlog for next spring.

Reply:
I agree with you 100%